content/blog/zen.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "The Zen of CoreDNS"
+description = "Philosophy behind CoreDNS"
+tags = ["Zen", "Meta"]
+draft = false
+date = "2019-09-28T13:55:25Z"
+author = "CoreDNS Authors"
++++
+
+CoreDNS is a DNS server, written in Go. It links *plugins* to provide a customized DNS service.
+This document details the philosophy behind it and provides guidance to future maintainers and
+users alike.
+
+# The Zen of CoreDNS
+
+Our main principle is that **we want to maintain an excellent implementation of a DNS server**.

```

Comment:
I know this came up in a tweet recently, but would be awesome if we had some level of conformance test to quantify 'excellent implementation'. Not an action item, just a comment.